release/README-maintainers.html

```diff
@@ -78,6 +79,8 @@
     Together, the release_build/release_push scripts take around 3 hours (including manual steps) when all goes according to plan.
     However, the most lengthy steps are automatic so you will be able to work on other tasks.
 </p>
+<p><strong>See also</strong> the <a href="README-development-process.html">Checker Framework Development Process</a> document which explains
```

Comment:
The development process document doesn't seem to be entirely up to date.  For example, the release person isn't responsible for triaging bugs any more.  It doesn't need to discuss things that don't exist any more like "recall class", and the "note on release numbers" should be taken care of in the release script or be in the release instructions.